sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientEncryptionKey.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.models.CosmosClientEncryptionKeyResponse;
+import com.azure.cosmos.util.Beta;
+
+/**
+ * The type Cosmos clientEncryptionKey. This contains methods to operate on a cosmos clientEncryptionKey
+ * synchronously
+ */
+@Beta(value = Beta.SinceVersion.V4_14_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
```

Comment:
Can you please update these Beta values to `V4_16_0` ?
You might need to add this `V4_16_0` to Beta.java enum.

Reply:
done